docs/products/opensearch/concepts/access_control.rst

```diff
@@ -1,23 +1,41 @@
 Access control
 ==============
 
-Aiven for OpenSearch® supports index-level access control lists (ACL) to control permissions. This approach allows you to limit the operations that are available to specific connections and to restrict access to certain data sets, which improves the security of your data.
+Aiven for OpenSearch® supports index-level access control lists (ACL) to control permissions.
+This approach allows you to limit the operations that are available to specific service users and
+restrict access to certain data sets.
+
+.. note::
+   When access control is initially enabled for the service, a set of rules that allows
+   full unlimited access to existing service users is created automatically.
+
+Rules are defined separately for each user as ``pattern/permission`` combinations. The ``pattern`` defines the indices that the permission applies to.
```

Comment:
I would put a subheading for pattern and one for permission to have it more clear for the user

Reply:
I'm not sure I understood where can we add this separations, it feels that the explanation uses both of them within a definition... I added **Patterns and permissions** subheader, does it make better? Or you can point out where to put the subheaders you had in mind!